src/store/traction/mutations.js

```diff
@@ -2,6 +2,12 @@ const mutations = {
     setTags(state, tags) {
         state.tractionTags = tags
     },
+    updateRun(state, run) {
+        let runIndex = state.pacbio.runs.runs.map(e => e.id).indexOf(run.id)
```

Comment:
I was wondering why this is runs.runs?? Not a criticism just looks a bit weird.

Reply:
Ye it does! It's just the heirarchy of the state folders, traction/pacbio/runs, then within the runs folder, there is a state called runs.